docs/COMMUNITY_PROJECTS.md

```diff
@@ -0,0 +1,31 @@
+# Community Projects
+
+This page celebrates the incredible work done by community members. 😊
+
+If you've made a project and would like to be featured here, open a
+pull request for this page!
+
+## Applications in the Wild
+
+* [Plain Text Accounting with TigerBeetle](https://github.com/pondersource/pta-tb-experiment#pta-tb-experiment)
+
+## Database Clients Made With ❤️  By the Community
+
+They are not officially supported by TigerBeetle but we do our best to
+support them anyway!
+
+Keep in mind before our first release there will be frequent breaking
+changes. We integration-test all first-party clients but third-party
+clients may lag behind.
+
+* [tigerbeetlex](https://github.com/rbino/tigerbeetlex): An Elixir client for TigerBeetle
+* [tigerbeetle-cpp](https://github.com/kassane/tigerbeetle-cpp): TigerBeetle C++ Client
+* [tigerbeetle-rs](https://github.com/ZetaNumbers/tigerbeetle-rs): Safe bindings to TigerBeetle
+
+Need a client for Go, Java, Node.js, or .NET? We've got you covered
+with [first-party
+clients](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients).
+
+## Other
+
+* [A Jepsen Test for TigerBeetle](https://github.com/nurturenature/jepsen-tigerbeetle)
```

Comment:
This could (with 50/50 probability) be moved under Applications in the Wild.

Reply:
Done!